From ca532ca6b977ecbecd1f8f83a04183c7878b4021 Mon Sep 17 00:00:00 2001
From: Zou Shunxiang <shunxian@codeaurora.org>
Date: Mon, 30 Oct 2017 11:19:09 +0800
Subject: [PATCH] net:ipv6: Disable dad to run ipv6 at early stage

With early ethernet scenario, we must disable dad to
get good KPI.

Change-Id: Ia8eae66d704e9e8b588ad12252c55e8f30e3812f
Signed-off-by: Zou Shunxiang <shunxian@codeaurora.org>
---
 net/ipv6/addrconf.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/ipv6/addrconf.c b/net/ipv6/addrconf.c
index 4b707ad..346f609 100644
--- a/net/ipv6/addrconf.c
+++ b/net/ipv6/addrconf.c
@@ -232,7 +232,7 @@ static struct ipv6_devconf ipv6_devconf __read_mostly = {
 	.proxy_ndp		= 0,
 	.accept_source_route	= 0,	/* we do not accept RH0 by default. */
 	.disable_ipv6		= 0,
-	.accept_dad		= 1,
+	.accept_dad		= 0,
 	.suppress_frag_ndisc	= 1,
 	.accept_ra_mtu		= 1,
 	.stable_secret		= {
@@ -280,7 +280,7 @@ static struct ipv6_devconf ipv6_devconf_dflt __read_mostly = {
 	.proxy_ndp		= 0,
 	.accept_source_route	= 0,	/* we do not accept RH0 by default. */
 	.disable_ipv6		= 0,
-	.accept_dad		= 1,
+	.accept_dad		= 0,
 	.suppress_frag_ndisc	= 1,
 	.accept_ra_mtu		= 1,
 	.stable_secret		= {
-- 
1.9.1

